go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/openziti/sdk-golang
 go 1.16
 
 //replace github.com/openziti/foundation => ../foundation
```

Comment:
You can't replace a project within itself. Can we drop this comment?

Reply:
@plorenz We can drop it. I left it in because I was troubleshooting building the project in a repo fork, something that I expect is typical for community contributors. I was able to build it with this `replace` directive, but did not wish to persist that change in the upstream. There's probably a better way.

Reply:
If I check out out your fork branch, it builds ok without that. I'm not clear on what problem it addressed, can you share some more details?